numbat/modules/datetime/human.nbt

```diff
@@ -12,7 +12,7 @@ fn _remove_plural_suffix(str: String) -> String =
   if str_slice(str, 0, 2) == "1 " then str_slice(str, 0, str_length(str) - 1) else str
 
 fn _human_seconds(dt: DateTime) -> String =
-  _remove_plural_suffix(format_datetime("%-S%.f seconds", dt))
```

Comment:
fractional seconds are currently not supported by jiff

Reply:
I can work on adding support for it. It was more of just limiting scope before the initial release more than anything else.

Reply:
=> https://github.com/BurntSushi/jiff/issues/54